web/components/Feed/Feed.tsx

```diff
@@ -306,11 +308,61 @@ function Feed(props: {
     }
   }
 
+  const allPosts = trpc.post.getAllPosts.useQuery();
+
   const [selectedFilters, setSelectedFilters] = useReducer(
     filterReducer,
     getInitialFilters()
   );
 
+  //const [queryFilters, setQueryFilters] = useReducer();
+  //const [displayedPosts, setDisplayedPosts] = useState(allPosts.data);
+
+  const formattedTestPosts = allPosts?.data?.map((p: IPost) => {
+    let date = p.date.toString();
+    date =
+      date.substring(0, date.indexOf("T")) +
+      " " +
+      date.substring(date.indexOf("T") + 1, date.indexOf(".") - 3);
+    date = date.replaceAll("-", "/");
+    if (parseInt(date.substring(11, 13)) > 12) {
+      date =
+        date.substring(0, 11) +
+        (parseInt(date.substring(11, 13)) - 12).toString() +
+        date.substring(13) +
+        " PM";
+    } else if (parseInt(date.substring(11, 13)) == 0) {
+      date = date.substring(0, 11) + "12" + date.substring(13) + " AM";
+    } else {
+      date = date + " AM";
+    }
```

Comment:
Use the `dayjs` library to format - it'll make this code more readable. When you do, double-check that it displays the date in the user's local time.